dds_web/api/project.py

```diff
@@ -436,12 +441,21 @@ def get(self):
                 # return ByteHours
                 project_info.update({"Usage": proj_bhours, "Cost": proj_cost})
 
-            project_info["Access"] = (
-                models.ProjectUserKeys.query.filter_by(
-                    project_id=p.id, user_id=current_user.username
-                ).count()
-                > 0
-            )
+            try:
+                project_info["Access"] = (
+                    models.ProjectUserKeys.query.filter_by(
+                        project_id=p.id, user_id=current_user.username
+                    ).count()
+                    > 0
+                )
+            except sqlalchemy.exc.OperationalError as err:
+                raise DatabaseError(
+                    message=str(err),
+                    alt_message="Database seems to be down."
+                    if isinstance(err, sqlalchemy.exc.OperationalError)
+                    else ".",
+                ) from err
```

Comment:
```suggestion
                raise DatabaseError(
                    message=str(err),
                    alt_message=(
                        "Could not get users project access information"
                        + (": Database malfunction." if isinstance(err, sqlalchemy.exc.OperationalError)
                        else "."),
                )
                ) from err
```
Same here with the linting